config/config_test.go

```diff
@@ -105,6 +105,7 @@ func emptyConfig() *Config {
 		SNS:                             ResourceType{FilterRule{}, FilterRule{}, "", false},
 		SQS:                             ResourceType{FilterRule{}, FilterRule{}, "", false},
 		SageMakerNotebook:               ResourceType{FilterRule{}, FilterRule{}, "", false},
+		SageMakerStudioDomain:           ResourceType{FilterRule{}, FilterRule{}, "", false},
```

Comment:
What was this added for? It seems unrelated to the rest of the changes.

Reply:
Hmm this may have been potentially missed in one of the previous PR? Need this change to pass all the tests.